config/stacks/fileManager.ts

```diff
@@ -21,6 +23,12 @@ export const getFileManagerStackProps = (stage: AppStage): FilemanagerConfig =>
     inventorySourceBuckets.push(fileManagerInventoryBucket[stage]);
   }
 
+  const eventSourceBuckets = [oncoanalyserBucket[stage], icav2PipelineCacheBucket[stage]];
+  if (stage == AppStage.PROD) {
```

Comment:
Might be worth putting in a comment here to say 'we only archive production data'